porch/engine/pkg/kpt/render.go

```diff
@@ -32,25 +37,35 @@ type renderer struct {
 var _ fn.Renderer = &renderer{}
 
 func (r *renderer) Render(ctx context.Context, pkg filesys.FileSystem, opts fn.RenderOptions) error {
-	rw := &kio.LocalPackageReadWriter{
-		PackagePath:        "/",
-		IncludeSubpackages: true,
-		FileSystem: filesys.FileSystemOrOnDisk{
-			FileSystem: pkg,
-		},
+	rr := render.Renderer{
+		PkgPath:    opts.PkgPath,
+		Runtime:    opts.Runtime,
+		FileSystem: pkg,
 	}
 
-	// Currently a noop rendering. TODO: Implement
-	nodes, err := rw.Read()
-	if err != nil {
-		return err
-	}
+	return rr.Execute(printer.WithContext(ctx, print{}))
+}
 
-	for _, n := range nodes {
-		ann := n.GetAnnotations()
-		ann["porch.kpt.dev/rendered"] = "yes"
-		n.SetAnnotations(ann)
-	}
+type print struct{}
+
+var _ printer.Printer = &print{}
+
+func (p print) PrintPackage(pkg *pkg.Pkg, leadingNewline bool) {
+
```

Comment:
Is this a debug implementation of printer?  A comment as to what the goal is here would be helpful (or maybe rename `print` to `debugPrinter` or similar)

Reply:
It is actually used (kpt renderer will fail without it) so I completed the implementation. It may turn out to be noisy (we'll see) in which case we can log at higher verbosity only